pkg/webhook/common.go

```diff
@@ -63,7 +56,8 @@ var (
 	serviceaccount                     = fmt.Sprintf("system:serviceaccount:%s:%s", util.GetNamespace(), serviceAccountName)
 	clientCAName                       = flag.String("client-ca-name", "", "name of the certificate authority bundle to authenticate the Kubernetes API server requests against")
 	certCNName                         = flag.String("client-cn-name", "kube-apiserver", "expected CN name on the client certificate attached by apiserver in requests to the webhook")
-	// webhookName is deprecated, set this on the manifest YAML if needed".
+	VwhName                            = flag.String("validating-webhook-configuration-name", "gatekeeper-validating-webhook-configuration", "name of the ValidatingWebhookConfiguration")
+	MwhName                            = flag.String("mutating-webhook-configuration-name", "gatekeeper-mutating-webhook-configuration", "name of the MutatingWebhookConfiguration")
```

Comment:
should we make a call out about these in the docs? maybe in a separate PR?

Reply:
Please could you point me to where exactly do you want me to document this?

Reply:
@acpana 

IMO it being available in `--help` and in the helm chart README is documentation enough.

We document how to run `--help` here:

https://open-policy-agent.github.io/gatekeeper/website/docs/customize-startup#other-configuration-options